sfge/src/test/java/com/salesforce/graph/source/supplier/InboundEmailHandlerSupplierTest.java

```diff
@@ -0,0 +1,46 @@
+package com.salesforce.graph.source.supplier;
+
+import com.salesforce.collections.CollectionUtil;
+import org.junit.jupiter.api.Test;
+
+public class InboundEmailHandlerSupplierTest extends BaseSourceSupplierTest {
+
+    /**
+     * Implementations of the {@code InboundEmailHandler#handleInboundEmail} method should be
+     * returned by this supplier.
+     */
+    @Test
+    public void supplierLoadsInboundEmailHandlerImplementations() {
```

Comment:
Equivalent to `PathEntryPointUtilTest#getPathEntryPoints_includesInboundEmailHandlerMethods` and `PathEntryPointUtilTest#testGetInboundEmailHandlerMethods`.